src/views/redirecting.vue

```diff
@@ -1,24 +1,196 @@
 <template>
-    <div class="weni-redirecting">
+  <div class="container">
+    <div v-if="loading" class="weni-redirecting">
         <loading />
     </div>
+    
+    <iframe
+      @load="onLoad"
+      v-show="!loading"
+      class="weni-redirecting"
+      allow="clipboard-read; clipboard-write;"
+      :src="src"
+      frameborder="0"
+    ></iframe>
+  </div>
 </template>
 
 <script>
 import Loading from '../components/Loading';
+import request from '../api/request.js';
+import Mgr from '../services/SecurityService';
+import axios from 'axios';
+
+const getRedirectUrls = async () => {
+  try {
+    const { uuid } = JSON.parse(localStorage.getItem('_project'));
+
+    return request.$http().get(`/v1/organization/project/${uuid}/`);
+  } catch(e) {
+    console.log('error', e);
+  }
+};
 
 export default {
-    name: 'Redirecting',
-    components: { Loading }
+  name: 'Redirecting',
+  components: { Loading },
+  data() {
+    return {
+      loading: false,
+      src: '',
+
+      urls: null,
+    };
+  },
+
+  async created() {},
+
+  watch: {
+    '$route.path': {
+      immediate: true,
+      handler () {
+        console.log('change path', this.$route.name);
+
+        if (this.$route.name === 'push') {
```

Comment:
`this.loading = true` can be before this if/else statement since it will always be true